tests/phpunit/api/v3/SyntaxConformanceTest.php

```diff
@@ -503,7 +503,6 @@ public static function toBeSkipped_updatesingle($sequential = FALSE) {
       'Relationship',
       'RelationshipType',
       'Note',
-      'OptionGroup',
```

Comment:
Just curious, why is `OptionGroup` being removed here?

Reply:
SO that when SyntaxConformanceTests run it won'y be skipped, This is an array of entities that are to be skipped when running a specific syntaxConformanceTest